api/src/main/scala/hmda/api/http/ValidationErrorConverter.scala

```diff
@@ -50,9 +50,17 @@ trait ValidationErrorConverter {
 
   val editDescriptions = EditMetaDataLookup.values
 
-  private def rowDetail(err: ValidationError): RowEditDetail = {
+  private def editDetail(err: ValidationError, vs: HmdaFileValidationState): EditResultRow = {
+    val row = relevantRow(err, vs)
+
+    if (err.ts) EditResultRow(RowId("Transmittal Sheet"), relevantFields(err, row))
```

Comment:
Here you are passing `err, row` which expands to `err, err, vs`. Therefor I think it might be cleaner to pass either only `row` or just `err, vs`.